client/src/components/onboarding/SetTimezoneUrl.js

```diff
@@ -1,20 +1,51 @@
-import React from "react";
+import React, { useState, useCallback } from "react";
 import {
   Grid,
   Typography,
   TextField,
   MenuItem,
   makeStyles,
-  Button,
+  Button
 } from "@material-ui/core";
 import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
+import axios from "axios";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import DoneIcon from "@material-ui/icons/Done";
+import ClearIcon from "@material-ui/icons/Clear";
+import {debounce} from "../../utils/utils";
 
-const SetTimezoneUrl = (props) => {
+const SetTimezoneUrl = props => {
   const classes = useStyles();
 
   const { url, setUrl, timezone, setTimezone } = props;
 
+  const [unique, setUnique] = useState(true);
+
+  
+
+  const checkUnique=async (linkVal)=>{
+    try {
+      const response = await axios.get(
+        `/api/user/is_unique?URL=${linkVal}`
```

Comment:
So if an error is returned then `setUnique = false` correct? Could the same thing be done where the get request returns a value of false?

Reply:
That's correct.  It can be done as you've suggested, but I think catch statement works a little better here as it will set unique=false if the call fails for any reason.